metadata.rb

```diff
@@ -9,6 +9,7 @@
 supports 'debian', '>= 6.0'
 supports 'ubuntu', '>= 12.04'
 supports 'redhat', '>= 6.3'
+supports 'centos', '>= 6.4'
```

Comment:
Should a `support 'fedora', '>- X.Y'` also be added?  Not sure fedora version vagrant requires.